kcbq-connector/src/main/java/com/wepay/kafka/connect/bigquery/config/BigQuerySinkConfig.java

```diff
@@ -346,7 +414,14 @@ private String sanitizeTableName(String tableName) {
    *         name if sanitization is enabled.
    */
   public String getTableFromTopic(String topic) {
-    return getBoolean(SANITIZE_TOPICS_CONFIG) ? sanitizeTableName(topic) : topic;
+    String tableName = null;
+    Map<String, String> map = getTopicsToTables();
```

Comment:
I suggest moving `getTablesToTopics` into SinkTask, since it's the only thing that uses it.